src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -25,22 +27,26 @@ export class FilterTimestamp extends HTMLElement {
 
   setFilterRange = (info) => {
     const selectedEl = browser.menus.getTargetElement(info.targetElementId);
-    const selectedRow = selectedEl.closest('tr');
+    if (!selectedEl) {
+      return;
+    }
```

Comment:
When we have multiple activity log page opened at a time, multiple context menu event listeners are being assigned but we get only one `selectedEl` (where the context event has been called)